openstack/cronus/sub-charts/maillog/templates/deployment.yaml

```diff
@@ -2,43 +2,26 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: maillog
-  {{- if .Values.annotations }}
-  annotations:
-    {{ toYaml .Values.annotations | indent 4 }}
-  {{- end  }}
-  {{- if .Values.labels }}
-  labels:
-    {{ toYaml .Values.labels | indent 4 }}
-  {{- end }}
 spec:
-  replicas: {{ default .Values.deployment.replicas }}
+  replicas: {{ .Values.deployment.replicas }}
   strategy:
-{{- if eq "RollingUpdate" .Values.updateStrategy }}
     rollingUpdate:
       maxSurge: {{ .Values.maxSurge }}
       maxUnavailable: {{ .Values.maxUnavailable }}
-{{- end }}
-    type: {{ .Values.updateStrategy }}
   selector:
     matchLabels:
       name: maillog
-      instance: maillog
   template:
     metadata:
       annotations:
         checksum/config: {{ include "maillog-config" . | sha256sum }}
       labels:
         name: maillog
-        instance: maillog
     spec:
-      serviceAccountName: maillog
```

Comment:
We removed the rbac, so I figured we could remove the service account. Maybe it is needed, but I think it is better to start without it and add when/if needed